yellow/false-positive-treatment.md

```diff
@@ -1,16 +1,24 @@
 # False Positive Treatment
 
-In order to not overlook real vulnerabilities, false positives are suppressed so they will not show up on the next tests again.
+Suppress false positives so they will not show up on the next tests again. This will help you to see the real vulnerabilities (not hidden among many many false positives anymore).
 
 For suppressions:
-- Document a reasonable justification
-- Practice review or pairing
+- Document a reasonable justification (use the locality principle: as close to the code as possible)
+    - Note on reasonable: Determine how much information your company needs, maybe it could be interesting for assessments as well?
+- Review your suppressions in a team (use [help](security-consulting-on-request.md) if necessary)
```

Comment:
```suggestion
- Document a reasonable justification (use the locality principle: as close to the code as possible).
    - Note on reasonable: Determine how much information your company needs, maybe it could be interesting for assessments as well?
- Review your suppressions in a team (use [help](security-consulting-on-request.md) if necessary).
```